prisma/migrations/20230406110409_postgres_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
```

Comment:
hey we should NOT need to replace the migration file like this.

if you actually have to make changes to migrations, make as minimal changes as possible in an additional migration (NEVER replace previous migrations) and submit it as a separate PR.  Changing the schema is a big deal. 

please NEVER replace an existing migration and do not include migrations with other changes, thanks